website/www/site/layouts/partials/header.html

```diff
@@ -92,3 +92,49 @@
       </ul>
     </div>
 </nav>
+
+<nav class="navigation-bar-desktop">
+  <a href={{ "/" | relLangURL }} class="navbar-logo" >
+    <img src="/images/beam_logo_navbar.png" alt="Beam Logo">
+  </a>
+  <div class="navbar-links">
+    <a class="navbar-link" href={{ "/get-started/beam-overview/" | relLangURL }}>{{ T "nav-get-started" }}</a>
+    <li class="dropdown navbar-dropdown navbar-dropdown-documentation">
+      <a class="navbar-link" href="#" class="dropdown-toggle" role="button" aria-haspopup="true" aria-expanded="false">
+        Documentation
+        <span>
+          {{ with resources.Get "icons/navbar-documentation-icon.svg" }}
+            {{ .Content | safeHTML }}
+          {{ end }}
+        </span>
```

Comment:
This renders strangely for me (on Desktop Chrome):
![image](https://user-images.githubusercontent.com/675055/100943010-3a7c1c00-34b1-11eb-9a69-64ec0c585f12.png)

Reply:
Ohh, could you please run it on private mode to make sure that there is no cache? I tested it on Chrome, Firefox, Safari and all fine.

Reply:
Weird I cannot make it go away by avoiding caching. Is it possible it only happens on the staged preview? Could you try looking at it through http://apache-beam-website-pull-requests.storage.googleapis.com/13439

Reply:
I can't access the link

<img width="778" alt="Screen Shot 2020-12-07 at 10 12 22" src="https://user-images.githubusercontent.com/23557826/101305241-ce186880-3874-11eb-909c-a36486a5dcfd.png">

Reply:
Anyway, I guess that I know the problem which occurs on staging. 🤔

Reply:
Ah sorry I think this is the link you need: http://apache-beam-website-pull-requests.storage.googleapis.com/13439/index.html
You can always find it by clicking details on the "Website Stage GCS" jenkins job: 
![image](https://user-images.githubusercontent.com/675055/101428433-91209600-38b5-11eb-955f-6ac3623d32b5.png)

That doesn't need to block this PR though